set.mm

```diff
@@ -507303,20 +507522,171 @@ codes is an increasing chain (with respect to inclusion).  (Contributed
+      DZUEZUIMZOZOZGPXSOZOZAHUFZOZBYDOZJZUGZCYDOZDYDOZJZUGZUHZHEKUJMZUKZXOXHGQJ
+      ZLZXPYAULXHXKYQXNXHYPGABUMMCDUMMUNIUOUPVBGEFQUQRXOYAGUAKUAUFXSOURZOZYCXOG
+      XTYRXOXHXRQJZLZXTYRULXHXKUUAXNXHYTXHXQQJYTEFUSXQNQUTRZVAZVBZUAXREFQVCRZVD
+      XOYRVEZPKJZGYBVFZJYSYCULXOUUFXTVEXOKVGOZYNVNZXTXOUUAUUIUUJXTVHUUDXREFQVIR
+      VJXOYRXTXOXTYRUUEVKVLVMUUGXOVOVPXOGPVGOZUUHXKXNGUUKJXHABCDGIVQVRXHXKUUHUU
+      KULZXNXHYTUUGUULUUBUUGXHVOVPXREPFQVSVTVBWAYRUAXSKPGYRWCWBWDWEXOYCYEYFXRSZ
+      UGZYIYJXRSZUGZUHZHYNUKZYOXHUUAXKXNYCUURULUUCXRABCDEFQGHIWFWGXOUUQYMHYNXOY
+      DYNJZLZUUNYHUUPYLUUTUUMYGUUMYEYFNSZYEYFXQSZLZUUTYGYEYFNXQWHYGUVAUUTUVCYEY
+      FBYDWIWMUUTUVBUVAUUTYEEJZYFEJZLZUVBXOUUSUVFXKXHUUSUVFTXNUUSXKUVFUUSXIUVDX
+      JUVEUUSKEYDVHZXIUVDTYDEKWJZUVGXIUVDKEAYDWKWNRUUSUVGXJUVETUVHUVGXJUVEKEBYD
+      WKWNRWLWOWPWQYEYFEEWRWSWTXAXBXDUUTUUOYKUUOYIYJNSZYIYJXQSZLZUUTYKYIYJNXQWH
+      YKUVIUUTUVKYIYJDYDWIWMUUTUVJUVIUUTYIEJZYJEJZLZUVJXOUUSUVNXNXHUUSUVNTXKUUS
+      XNUVNUUSXLUVLXMUVMUUSUVGXLUVLTUVHUVGXLUVLKECYDWKWNRUUSUVGXMUVMTUVHUVGXMUV
+      MKEDYDWKWNRWLWOXCWQYIYJEEWRWSWTXAXBXDXEXFWEXG $.
+  $}
+
+  ${
+    $d Z x $.
+    ex-sategoelelomsuc.s $e |- S = ( x e. _om
+                                     |-> if ( x = 2o , Z , suc Z ) ) $.
+    $( Example of a valuation of a simplified satisfaction predicate over the
+       ordinal numbers as model for a Godel-set of membership using the
+       properties of a successor: ` ( S `` 2o ) = Z e. suc Z = ( S `` 2o ) ` .
+       Remark: the indices ` 1o ` and ` 2o ` are intentionally reversed to
+       distinguish them from elements of the model: ` ( 2o e.g 1o ) ` should
+       not be confuse with ` 2o e. 1o ` , which is false.  (Contributed by AV,
+       19-Nov-2023.) $)
+    ex-sategoelelomsuc $p |- ( Z e. _om -> S e. ( _om SatE ( 2o e.g 1o ) ) ) $=
+      ( com wcel c2o c1o co cfv wceq cvv omex adantl 2onn fvmptd 1onn wa pm3.2i
+      a1i cgoe csate cmap wf cv csuc id peano2 ifcld adantr fmptd elmapd mpbird
+      cif sucidg cmpt iftrue 1one2o neii mtbiri iffalsed 3eltr4d wb sategoelfvb
+      eqeq1 eqid mp1i mpbir2and ) CEFZBEGHUAIUBIZFZBEEUCIFZGBJZHBJZFZVIVLEEBUDV
+      IAEAUEZGKZCCUFZUNZEBVIVSEFVPEFVIVQCVREVIUGZCUHZUIUJDUKVIEEBLLELFZVIMTZWCU
+      LUMVICVRVMVNCEUOVIAGVSCEBEBAEVSUPKVIDTZVQVSCKVIVQCVRUQNGEFZVIOTVTPVIAHVSV
+      REBEWDVPHKZVSVRKVIWFVQCVRWFVQHGKHGURUSVPHGVEUTVANHEFZVIQTWAPVBWBWEWGRZRVK
+      VLVORVCVIWBWHMWEWGOQSSGHBVJELVJVFVDVGVH $.
+  $}
+
+  ${
+    ex-sategoelel12.s $e |- S = ( x e. _om |-> if ( x = 2o , 1o , 2o ) ) $.
+    $( Example of a valuation of a simplified satisfaction predicate over a
+       proper pair (of ordinal numbers) as model for a Godel-set of membership
+       using the properties of a successor:
+       ` ( S `` 2o ) = 1o e. 2o = ( S `` 2o ) ` .  Remark: the indices ` 1o `
+       and ` 2o ` are intentionally reversed to distinguish them from elements
+       of the model: ` ( 2o e.g 1o ) ` should not be confuse with
```

Comment:
```suggestion
       of the model: ` ( 2o e.g 1o ) ` should not be confused with
```
in retrospect these typo comments are unimportant and nonblocking; in fact I may do a pr soon regarding df-ich so such things may be fixed there

Reply:
Typos fixed.